recipes/dedupe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "dedupe" %}
+{% set version = "2.0.8" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dedupe-{{ version }}.tar.gz
+  sha256: f360c5a3ac940623f00971c93836213147b826b7f87bbb81ce9e7ff73c67847a
+
+build:
+  number: 0
+  noarch: python
```

Comment:
This one requires a compiler so it cannot be noarch.

Reply:
@ocefpaf Okay I have removed the noarch and instead chose to skip the win build due to some of the dedupe dependencies not being available on windows.

Reply:
No need to skip b/c that will give you a linter error asking you to remove noarch. Let Windows fail here due to the missing dep and it is OK.